backup-postgres/action.yml

```diff
@@ -23,6 +23,12 @@ inputs:
   backup-file:
     description: Name of the backup file
     required: true
+  keyvault-name:
```

Comment:
The calling workflow may have the webhook in github secret or keyvault, or the secret may not always be called the same.
I think it should be the reponsibility of the calling workflow to retrieve the webhok and pass it as input.

Reply:
ok, which one do we want to use in the workflow template? keyvault (I assume) or github secret?

Reply:
Yes, it makes sense to use keyvault in the template
And pass `slack-webhook` as input here

Reply:
updated